frontend/src/App.tsx

```diff
@@ -136,12 +136,19 @@ export default class App extends React.Component {
   }
 
   private async updateMetadata() {
-    const { uniqueAddressCount, spacePledged } = await fetchMetadata();
-
-    this.appUpdate({
-      spacePledged: parseInt(spacePledged, 10),
-      uniqueAddressCount: parseInt(uniqueAddressCount, 10),
-    });
+    try {
+      const { uniqueAddressCount, spacePledged } = await fetchMetadata();
+
+      this.appUpdate({
+        spacePledged: parseInt(spacePledged, 10),
+        uniqueAddressCount: parseInt(uniqueAddressCount, 10),
+      });
+
+      setTimeout(async () => await this.updateMetadata(), 10000);
```

Comment:
I think we need to make next attempt regardless of whether previous request failed. If server is unable to respond temporarily it doesn't mean server is permanently unavailable.

Reply:
yes, my bad